### PR TITLE
[bump] protocol-definitions: 1.1.0 => 1.2.0 (minor)

### PR DESCRIPTION
**Pull request by tylerbutler**
Bumped protocol-definitions from 1.1.0 to 1.2.0 since 1.1.0 has been released.

Commands run:

```shell
flub bump @fluidframework/protocol-definitions -t minor
cd common/lib/protocol-definitions
npm run build
git add -A && git commit -m "typetests"
```

### CODE REVIEW

File: common/lib/protocol-definitions/package.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
     "typescript-formatter": "7.1.0"
   },
   "typeValidation": {
-    "version": "1.1.0",
+    "version": "1.2.0",
```
Comment (CraigMacomber) [inline common/lib/protocol-definitions/package.json:63]: This seems like an unrelated change. is it related somehow?

Comment (tylerbutler) [reply common/lib/protocol-definitions/package.json:63]: I think that when the version is umped the typeValidation version should also be bumped. That said, it's inconsistent to do it in this PR -- I'll remove this and come back through with typetest updates as needed.

Comment (tylerbutler) [inline common/lib/protocol-definitions/package.json:63]:
```suggestion
    "version": "1.1.0",
```

**Review by CraigMacomber — APPROVED** [review]

**Comment by tylerbutler** [comment]
@CraigMacomber So in a real facepalm moment, the CI was failing because I failed to run the ONE TRUE BUILD COMMAND locally which includes the typetest gen. SIGH. Anyway, I commiserate with your frustrations about the inconsistency and inscrutability of our CI/build system.

**Comment by tylerbutler** [comment]
> Putting what command you ran to generate the changes in the PR description would improve discoverability of both the one true build command, and reviewers' ability to detect when you didn't do it. I think its a great practice when changes were done in an automated way.

Great idea, and done in this PR.

In this case, the build command was... `build`. I just didn't do a local one after running npm i and updating the lockfiles. I'll file a follow up issue on the release tools to consider running a build locally as part of the bump. The point of the automation is to make it harder to forget this kind of thing.
